tests_pytest/pytorch_tests/e2e_tests/post_training_quantization_tests/test_quantization_error_methods.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2025 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import math
+
+import numpy as np
+import pytest
+import torch
+from mct_quantizers import QuantizationMethod
+from torch import nn
+
+from model_compression_toolkit.core import QuantizationErrorMethod, QuantizationConfig, CoreConfig
+from model_compression_toolkit.core.pytorch.utils import to_torch_tensor
+from model_compression_toolkit.ptq import pytorch_post_training_quantization
+from model_compression_toolkit.target_platform_capabilities.schema.mct_current_schema import OpQuantizationConfig, \
+    AttributeQuantizationConfig, Signedness
+from model_compression_toolkit.target_platform_capabilities.constants import KERNEL_ATTR, BIAS_ATTR
+from tests.common_tests.helpers.tpcs_for_tests.v4.tpc import generate_tpc
+
+INPUT_SHAPE = (1, 8, 12, 8)
+torch.manual_seed(42)
+
+
+def build_model(input_shape, out_chan, kernel, const):
```

Comment:
this is a very basic model, maybe add one or two models with more "complicated" activation layers?